<launch>
  <arg name="numBoards" default="20" />
  <arg name="numCornersHor" default="8"/>
  <arg name="numCornersVer" default="6"/>
  <arg name="frame_rate" default="30" />
  <arg name="camera" default="forward" />

  <include file="$(find riptide_hardware)/launch/cameras.launch" />

  <node pkg="riptide_hardware" type="calibrate_camera" name="calibrate_camera" output="screen">
    <param name="numBoards" type="double" value="$(arg numBoards)" />
    <param name="numCornersHor" type="double" value="$(arg numCornersHor)" />
    <param name="numCornersVer" type="double" value="$(arg numCornersVer)" />
    <param name="frame_rate" type="double" value="$(arg frame_rate)" />
    <param name="camera" type="string" value="$(arg camera)" />
  </node>
</launch>
